src/frontend/src/components/Login/Login.css

```diff
@@ -1,10 +1,3 @@
 .avatar {
```

Comment:
Can we move the `avatar` styles into your JS as well and delete this file?

Reply:
Changes made